tests/web-server/index.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, it, before, after } from 'node:test';
+import * as assert from 'node:assert';
```

Comment:
One thing we should consider when as we start developing v2, is the possibility to hold tests alongside code, e.g. web-server tests in the same folder as the web-server components.
This is just a consideration for future and does not have much to do with this PR.

Reply:
The challenge is that we need to deploy the stack to get actual outputs. This influences how we group and design tests, and the setup.
I suggest we keep this for now, and as we add more tests, especially E2E ones, reconsider the decision.
This will provide us more insight into the implications by the time we make the change/decision.

Reply:
That definitely makes sense.